flytekit/types/schema/types.py

```diff
@@ -320,7 +320,7 @@ class FlyteSchemaTransformer(TypeTransformer[FlyteSchema]):
         _np.float32: SchemaType.SchemaColumn.SchemaColumnType.FLOAT,
         _np.float64: SchemaType.SchemaColumn.SchemaColumnType.FLOAT,
         float: SchemaType.SchemaColumn.SchemaColumnType.FLOAT,
-        _np.bool: SchemaType.SchemaColumn.SchemaColumnType.BOOLEAN,  # type: ignore
+        _np.bool_: SchemaType.SchemaColumn.SchemaColumnType.BOOLEAN,  # type: ignore
```

Comment:
Why have you changed this? Any specific reason?

Reply:
While running tests, numpy was giving warning:
```
`np.bool` is a deprecated alias for the builtin `bool`.  If you specifically wanted the numpy scalar type, use `np.bool_` here.
```
so have updated it